Client/Frontend/Widgets/Menu/MenuView.swift

```diff
@@ -342,6 +342,8 @@ class MenuView: UIView {
             }
             toolbar.setItems(displayToolbarItems, animated: false)
         }
+        toolbar.backgroundColor = toolbarColor
+        toolbar.setNeedsDisplay()
```

Comment:
Nit: I think setting the contentMode to `.Redraw` should work instead of manually calling `setNeedsDisplay`. This way we can guarantee that we will redraw on every bounds change.